src/transform/tests/test_transforms/anf.spec

```diff
@@ -159,76 +159,66 @@ With
         Get t0
 ----
 Return
-  Get l20
+  Get l21
```

Comment:
The comment above this test has become outdated. As it was documenting the shortcomings that your PR is solving, you could simply delete the comment.